app/assets/javascripts/standalone/react_components.js

```diff
@@ -0,0 +1,3 @@
+//= require react
```

Comment:
Renamed this file to reflect that it's loading react components, easy to find and see which pages are loading react and which aren't.